src/Serilog.Sinks.Datadog/Serilog.Sinks.Datadog.csproj

```diff
@@ -1,78 +1,16 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{F3B80F77-155C-434D-9865-2E522D072972}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Serilog</RootNamespace>
+    <TargetFrameworks>netstandard2.0;netstandard1.3;net451</TargetFrameworks>    
     <AssemblyName>Serilog.Sinks.Datadog</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <PackageId>Serilog.Sinks.Datadog</PackageId>
+    <Version>2.0.0</Version>
```

Comment:
This needs to be a `<VersionPrefix>` and since we have 2.0.8 out there, might be good to rev the version number to 3.0.0.